GameOfLife/GameOfLife/Controllers/InputController.cs

```diff
@@ -1,19 +1,22 @@
 ﻿using GameOfLife.Interfaces;
 using GameOfLife.Models;
 using static GameOfLife.StringConstantsModel;
+using static GameOfLife.Views.MenuViews;
```

Comment:
I would advise against adding these `using static`, as it makes the code really hard to read/understand, since its not clear, when you reference a method/property from these static classes in code, from where exactly it is coming from (At least its not clear for someone like me, who is not familiar with the code base)

Reply:
Good point, haven't even thought about it that way. Replaced 'using static {...}' with 'MenuViews.{...}' and 'StringConstants.{...}'.